libs/net/elastic/TNOElasticClient.cs

```diff
@@ -75,7 +75,7 @@ private static ConnectionSettings GetConnectionSettings(ElasticOptions options)
         public async Task<SearchResultModel<T>> SearchAsync<T>(string index, JsonDocument query)
             where T : class
         {
-            var url = this.Options.Url!.Append($"/{index}/_search?pretty=true&typed_keys=true");
+            var url = this.Options.Url!.Append($"/{index}/_search?pretty=true&typed_keys=true&sort=publishedOn:desc");
```

Comment:
How about making it overridable?  Add an option argument.

Reply:
Updated, thanks